src/qbittorrent_telegram_notification.sh

```diff
@@ -1,23 +1,40 @@
 #!/bin/bash
 
-# Bot token
-BOT_TOKEN="ADD_YOUR_TELEGRAM_BOT_TOKEN"
+# Source the configuration file for BOT_TOKEN and CHAT_ID
+BASEDIR=$(dirname "$0")
+source "${BASEDIR}/bot_creds.sh"
 
-# Your chat id
-CHAT_ID="ADD_YOUR_TELEGRAM_CHAT_ID"
-
-# Get variables from qbittorrent
+# Extract specific arguments if needed
 TORRENT_NAME="$1"
+TORRENT_SIZE_BYTES="$2"
+
+# Convert size from bytes to MB or GB as appropriate
+if [ $TORRENT_SIZE_BYTES -lt 1073741824 ]; then
+    # If size is less than 1 GB, convert to MB
+    TORRENT_SIZE=$(echo "scale=2; $TORRENT_SIZE_BYTES/1048576" | bc)
+    SIZE_UNIT="MB"
+else
+    # Otherwise, convert to GB
+    TORRENT_SIZE=$(echo "scale=2; $TORRENT_SIZE_BYTES/1073741824" | bc)
+    SIZE_UNIT="GB"
+fi
 
 # Notification message
 # If you need a line break, use "%0A" instead of "\n".
-MESSAGE="%3Cstrong%3EDownload%20Completed%3C%2Fstrong%3E%0A${TORRENT_NAME}%0A"
+MESSAGE="✅ ${TORRENT_NAME} [${TORRENT_SIZE} ${SIZE_UNIT}]" 
```

Comment:
I'm OK with this, but could you keep the `Download completed` text? For people who use the same chat for multiple bots, it could be handy to have that text at the beginning of the message. That's the reason I've added that text to the first line of the message. What do you think about it?

Reply:
Updated in latest version of PR. 
Added a message variable definition, to make it cleaner